pdp/portals/bcsd_downscale_archive.py

```diff
@@ -1,6 +1,8 @@
-'''The pdp.portals.bcsd_downscale_canada module configures a raster
-portal to serve BCSD and BCCAQ downscaled (10km) data over all of
-Canada.
+'''This portal serves the version 1 BCCAQ downscaled (10km) data
```

Comment:
While we're renaming this file, can we remove "bcsd" from the name (since it contains both BCSD *and* BCCAQ data). How about ``downscale_archive.py``?